--- nl.before
+++ nl.candidate
@@ -1,42 +1,19 @@
 <div id="page-content">
-<script>// <![CDATA[
-function remove_classes_active() {
-    alert( 'FOUND' );
-    var $aSelected = $('.question-answer-block').find('a');
-    
-    if( $aSelected.hasClass('active') ){
-      alert( 'FOUND' );
-      $('.question-answer-block > a').removeClass('active');
-    }else{
-      alert( 'NOT FOUND' );
-    }
-  }
-// ]]></script><script>// <![CDATA[
-jQuery("document").ready(function($){
-    var nav = $('#nav');
-    $(window).scroll(function () {
-     	if(getWidthBrowser() > 1200){
-    		if ($(this).scrollTop() > 136){
-              nav.addClass("on animated"); 		
-            } 
-            else {
-              nav.removeClass("on animated"); 		
-            } 
-  		}
-          $('#goGrid').trigger('click');
+<style>
+#page-content .help-page summary.ask { display: list-item; list-style: disclosure-closed inside; min-height: 44px; padding: 10px 0; cursor: pointer; }
+#page-content .help-page details[open] > summary.ask { list-style-type: disclosure-open; }
+#page-content .help-page summary.ask::-webkit-details-marker { display: inline; }
+#page-content .help-page summary.ask h3 { display: inline; margin: 0; }
+#page-content .help-page summary.ask:focus-visible { outline: 2px solid currentColor; outline-offset: 4px; }
+</style>
 
-      
-    });  
-    
-  });
-// ]]></script>
 <div class="help-page">
 <div id="General Questions">
 <div class="faq-item-title" name="General Questions">
 <h2>Over ons</h2>
-<div class="question-answer-block">
-<h3><a class="ask" onclick="if (jQuery(this).parent().next('div').css('display') == 'block') { jQuery('.answer').slideUp(); $(this).removeClass('active');}else{ jQuery('.answer').slideUp(); jQuery(this).parent().next('div').slideDown(); $('.active').removeClass('active'); $(this).addClass('active');} return false;">A. Algemene vragen</a></h3>
-<div class="answer" style="display: none; font-size: 16px;">
+<details class="question-answer-block" name="dlm-faq">
+<summary class="ask"><h3>A. Algemene vragen</h3></summary>
+<div class="answer" style="font-size: 16px;">
 <p><strong>1. Is deze site veilig in gebruik? </strong></p>
 <p>Dat is het zeker. We gebruiken PayPal om betalingen te verwerken. Dus zelfs ons personeel ziet je betalingsgegevens niet. PayPal is de grootste beveiligde betalingsprovider ter wereld. Onze website is gebouwd op het beveiligde Shopify-platform. Al je persoonlijke informatie is goed beveiligd.</p>
 <p> </p>
@@ -48,36 +25,36 @@
 <p> </p>
 <p><strong>4. Waarom zijn jullie zo goedkoop?</strong></p>
 <p>We schakelen alle tussenpersonen uit en leveren de producten rechtstreeks aan consumenten van onze fabrikanten, waardoor we extreem lage prijzen kunnen bieden.</p>
-</div>
-</div>
+</div>
+</details>
 </div>
 <div id="Before You Place an Order">
 <div class="faq-item-title" name="before you place an order">
 <h2>Voordat je een bestelling plaatst</h2>
 </div>
-<div class="question-answer-block">
-<h3><a class="ask" onclick="if (jQuery(this).parent().next('div').css('display') == 'block') { jQuery('.answer').slideUp(); $(this).removeClass('active');}else{ jQuery('.answer').slideUp(); jQuery(this).parent().next('div').slideDown(); $('.active').removeClass('active'); $(this).addClass('active');} return false;">A. Maten</a></h3>
-<div class="answer" style="display: none; font-size: 16px;">
+<details class="question-answer-block" name="dlm-faq">
+<summary class="ask"><h3>A. Maten</h3></summary>
+<div class="answer" style="font-size: 16px;">
 <p><strong>1. Waar kan ik de maattabel in centimeters vinden?</strong></p>
 <p>Je kunt maattabellen in centimeters vinden op onze maattabelpagina hier: <a href="http://www.assassinshoodies.com/pages/sizing-information" alt="Maattabelpagina">Maattabelpagina</a> of in de beschrijving van elk product.</p>
 <p> </p>
 <p><strong>2. Krijg ik een artikel dat twee maten groter is?</strong></p>
 <p>Nee. We sturen de maat die je hebt besteld. Als je normaal gesproken Europese medium maat draagt, kies dan medium (M) maat en je ontvangt een medium maat.</p>
-</div>
-</div>
-<div class="question-answer-block">
-<h3><a class="ask" onclick="if (jQuery(this).parent().next('div').css('display') == 'block') { jQuery('.answer').slideUp(); $(this).removeClass('active');}else{ jQuery('.answer').slideUp(); jQuery(this).parent().next('div').slideDown(); $('.active').removeClass('active'); $(this).addClass('active');} return false;">B. Betalingen</a></h3>
-<div class="answer" style="display: none; font-size: 16px;">
+</div>
+</details>
+<details class="question-answer-block" name="dlm-faq">
+<summary class="ask"><h3>B. Betalingen</h3></summary>
+<div class="answer" style="font-size: 16px;">
 <p><strong>1. Welke betaalmethoden accepteren jullie?</strong></p>
 <p>We accepteren PayPal en creditcards/debetkaarten (Visa, Mastercard, American Express, Discover).</p>
 <p> </p>
 <p><strong>2. Ik wil met mijn creditcard betalen, maar ik word doorgestuurd naar de PayPal-pagina?</strong></p>
 <p>We gebruiken PayPal om al onze betalingen te verwerken. Op de PayPal-pagina rechts kun je kiezen uit twee opties: "Betalen met mijn PayPal-account" <i>of</i> "Betalen met mijn creditcard of betaalpas"</p>
-</div>
-</div>
-<div class="question-answer-block">
-<h3><a class="ask" onclick="if (jQuery(this).parent().next('div').css('display') == 'block') { jQuery('.answer').slideUp(); $(this).removeClass('active');}else{ jQuery('.answer').slideUp(); jQuery(this).parent().next('div').slideDown(); $('.active').removeClass('active'); $(this).addClass('active');} return false;">C. Verzending</a></h3>
-<div class="answer" style="display: none; font-size: 16px;">
+</div>
+</details>
+<details class="question-answer-block" name="dlm-faq">
+<summary class="ask"><h3>C. Verzending</h3></summary>
+<div class="answer" style="font-size: 16px;">
 <p><strong>1. Wat is de verzendprijs?</strong></p>
 <p>We bieden absoluut GRATIS levering wereldwijd.</p>
 <p> </p>
@@ -89,16 +66,16 @@
 <p> </p>
 <p><strong>4. Moet ik douanekosten betalen?</strong></p>
 <p>Nee, je hoeft geen douanekosten te betalen. We markeren al onze bestellingen als cadeaus met lage waarde. Bovendien sturen we grotere bestellingen in aparte pakketten.</p>
-</div>
-</div>
+</div>
+</details>
 </div>
 <div id="After You Place an Order">
 <div class="faq-item-title" name="After You Place an Order">
 <h2>Nadat je een bestelling hebt geplaatst</h2>
 </div>
-<div class="question-answer-block">
-<h3><a class="ask" onclick="if (jQuery(this).parent().next('div').css('display') == 'block') { jQuery('.answer').slideUp(); $(this).removeClass('active');}else{ jQuery('.answer').slideUp(); jQuery(this).parent().next('div').slideDown(); $('.active').removeClass('active'); $(this).addClass('active');} return false;">A. Orderstatus</a></h3>
-<div class="answer" style="display: none; font-size: 16px;">
+<details class="question-answer-block" name="dlm-faq">
+<summary class="ask"><h3>A. Orderstatus</h3></summary>
+<div class="answer" style="font-size: 16px;">
 <p><strong>1. Wat betekent de orderstatus "Onvervuld"?</strong></p>
 <p>Je bestelling wordt verwerkt en het trackingnummer van de bestelling is nog in afwachting. We verwerken de bestellingen meestal en sturen onze klanten het trackingnummer zodra het trackingnummer aan het postsysteem is doorgegeven, wat 5-7 dagen duurt nadat de bestelling al is verzonden.</p>
 <p> </p>
@@ -107,11 +84,11 @@
 <p> </p>
 <p><strong>3. Het duurt te lang, waar is mijn bestelling?</strong></p>
 <p>Controleer alstublieft de tracking van uw bestelling voor meer informatie. Als het probleem blijft bestaan, stuur dan een e-mail naar: <a href="mail%20to:support@dresslikemommy.com">support@dresslikemommy.com</a></p>
-</div>
-</div>
-<div class="question-answer-block">
-<h3><a class="ask" onclick="if (jQuery(this).parent().next('div').css('display') == 'block') { jQuery('.answer').slideUp(); $(this).removeClass('active');}else{ jQuery('.answer').slideUp(); jQuery(this).parent().next('div').slideDown(); $('.active').removeClass('active'); $(this).addClass('active');} return false;">B. Ordertracking</a></h3>
-<div class="answer" style="display: none; font-size: 16px;">
+</div>
+</details>
+<details class="question-answer-block" name="dlm-faq">
+<summary class="ask"><h3>B. Ordertracking</h3></summary>
+<div class="answer" style="font-size: 16px;">
 <p><strong>1. Hoe kan ik mijn bestelling volgen?</strong></p>
 <p>Wij verstrekken de trackingcode van de bestelling 5-7 dagen nadat u een bestelling heeft geplaatst. U kunt uw bestelling hier volgen: <a hef="www.assassinshoodies.com/pages/track-your-order" alt="Volg uw bestelling">Volg uw bestelling</a></p>
 <p> </p>
@@ -123,11 +100,11 @@
 <p> </p>
 <p><strong>4. Hoe blijf ik op de hoogte van mijn ordertracking?</strong></p>
 <p>Wij gebruiken het After Ship-tracking systeem om de bestellingen van onze klanten te volgen, dus wanneer de trackinginformatie wordt bijgewerkt, ontvangt u een e-mail.</p>
-</div>
-</div>
-<div class="question-answer-block">
-<h3><a class="ask" onclick="if (jQuery(this).parent().next('div').css('display') == 'block') { jQuery('.answer').slideUp(); $(this).removeClass('active');}else{ jQuery('.answer').slideUp(); jQuery(this).parent().next('div').slideDown(); $('.active').removeClass('active'); $(this).addClass('active');} return false;">B. Ik wil mijn bestelling wijzigen of annuleren</a></h3>
-<div class="answer" style="display: none; font-size: 16px;">
+</div>
+</details>
+<details class="question-answer-block" name="dlm-faq">
+<summary class="ask"><h3>B. Ik wil mijn bestelling wijzigen of annuleren</h3></summary>
+<div class="answer" style="font-size: 16px;">
 <p><strong>1. Ik heb iets besteld, maar ik wil annuleren</strong></p>
 <p>U moet binnen 1 werkdag na het plaatsen van de bestelling contact met ons opnemen om een wijziging te regelen. Dit kunt u doen door een e-mail te sturen naar <a href="mail%20to:support@dresslikemommy.com">support@dresslikemommy.com</a> met uw bestelnummer als onderwerp.</p>
 <p> </p>
@@ -136,16 +113,16 @@
 <p> </p>
 <p><strong>3. Ik heb iets besteld, maar wil het afleveradres wijzigen</strong></p>
 <p>U moet binnen 1 werkdag na het plaatsen van de bestelling contact met ons opnemen om een wijziging te regelen. Dit kunt u doen door een e-mail te sturen naar <a href="mail%20to:support@dresslikemommy.com">support@dresslikemommy.com</a> met uw nieuwe afleveradres.</p>
-</div>
-</div>
+</div>
+</details>
 </div>
 <div id="After You Receive an Order">
 <div class="faq-item-title" name="After You Receive an Orde">
 <h2>Nadat u een bestelling heeft ontvangen</h2>
 </div>
-<div class="question-answer-block">
-<h3><a class="ask" onclick="if (jQuery(this).parent().next('div').css('display') == 'block') { jQuery('.answer').slideUp(); $(this).removeClass('active');}else{ jQuery('.answer').slideUp(); jQuery(this).parent().next('div').slideDown(); $('.active').removeClass('active'); $(this).addClass('active');} return false;">A. Ik wil mijn bestelling wijzigen of retourneren</a></h3>
-<div class="answer" style="display: none; font-size: 16px;">
+<details class="question-answer-block" name="dlm-faq">
+<summary class="ask"><h3>A. Ik wil mijn bestelling wijzigen of retourneren</h3></summary>
+<div class="answer" style="font-size: 16px;">
 <p><strong>1. Mijn bestelling is aangekomen, maar de artikelen waren twee maten te groot</strong></p>
 <p>Let op dat alle maten in onze winkel worden weergegeven in EU- en US-maateenheden. De daadwerkelijke kledingmaat (op het product vermeld) is meestal twee keer zo groot. Onze fabrikanten bevinden zich in Oost-Azië en wij sturen automatisch de Aziatische maat die overeenkomt met de Europese/US-maat. Dat betekent dat als u Medium (M – EU/US) bestelt, wij u Extra Large (XL) Aziatische equivalent sturen.</p>
 <p> </p>
@@ -157,8 +134,8 @@
 <p> </p>
 <p><strong>4. Mijn bestelling is aangekomen, maar ik wil het retourneren.</strong></p>
 <p>Neem alstublieft contact met ons op via <a href="mail%20to:support@dresslikemommy.com">support@dresslikemommy.com</a> en geef ons uw bestelnummer door. U kunt het artikel naar ons terugsturen en wij zullen het aankoopbedrag terugbetalen. *Verzendkosten zijn voor uw rekening.</p>
-</div>
-</div>
+</div>
+</details>
 </div>
 </div>
 </div>
